metrics/github/repos.py

```diff
@@ -1,4 +1,17 @@
-NON_TECH_REPOS = {
+from metrics.github import query
+
+
+def our_repos(client, org):
+    return list(filter(_is_tech_owned, query.repos(client, org)))
```

Comment:
My strong feeling about list comprehensions is that they're not always the best solution and sometimes higher-order functions are cleaner. I think the list comprehension is actually better here, though, since Python's insistence on iterator-maximalism has forced me to wrap an extra list round the `filter()` call.

Thank you.